server/Dockerfile

```diff
@@ -1,23 +1,24 @@
-# This stage is tagged for local development.
+FROM curlimages/curl AS download
+WORKDIR /var/tmp
+RUN curl -O https://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/liftOver \
+    -O https://hgdownload.cse.ucsc.edu/goldenpath/hg19/liftOver/hg19ToHg38.over.chain.gz \
+    -O https://hgdownload.cse.ucsc.edu/goldenpath/hg38/liftOver/hg38ToHg19.over.chain.gz && \
+    gunzip ./hg38ToHg19.over.chain.gz && \
+    gunzip ./hg19ToHg38.over.chain.gz
+
 FROM node:16-slim AS dev
 WORKDIR /usr/src/app
-RUN apt-get update && apt-get install -y wget libcurl4
-RUN wget -O /usr/local/bin/liftOver https://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/liftOver  \
-  && chmod +x /usr/local/bin/liftOver
-COPY . .
-WORKDIR /home/node
-RUN mkdir chains && \
-  wget -O ./chains/hg19ToHg38.over.chain.gz https://hgdownload.cse.ucsc.edu/goldenpath/hg19/liftOver/hg19ToHg38.over.chain.gz && \
-  wget -O ./chains/hg38ToHg19.over.chain.gz https://hgdownload.cse.ucsc.edu/goldenpath/hg38/liftOver/hg38ToHg19.over.chain.gz && \
-  gunzip ./chains/hg38ToHg19.over.chain.gz && \
-  gunzip ./chains/hg19ToHg38.over.chain.gz
-RUN chown -R node:node ./ && chown -R node:node /home/node
-WORKDIR /usr/src/app
+COPY --from=download /var/tmp/liftOver /usr/local/bin 
+RUN chmod +x /usr/local/bin/liftOver && apt-get update && apt-get install -y libcurl4
+COPY --from=download --chown=node:node /var/tmp/*.over.chain /home/node/
```

Comment:
You should use this dev stage as the base layer for the next one so these downloads are cached as the lowest layer we add.